openstack/kmip/templates/ingress.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     heritage: "{{ .Release.Service }}"
     system: openstack
     type: api
-    component: barbican
+    component: kmip
   annotations:
   {{- if .Values.tlsacme }}
     kubernetes.io/tls-acme: "true"
```

Comment:
missing disco here

Reply:
its not showing in the diff strangely :

https://github.com/sapcc/helm-charts/blob/7ae4d3aacfaed4848b3b5fd0a5fe2bcb03ec3139/openstack/kmip/templates/ingress.yaml#L18